README.md

```diff
@@ -115,6 +116,81 @@ testing. To test:
 
 Testing is only supported on Linux with bash.
 
+## Post-Quantum Algorithms
+
+Starting with wolfSSL version 5.0.1 and nginx version 1.21.4, You can now enable the integration of liboqs in wolfSSL thus enabling post-quantum algorithms for your HTTPS connections over TLS 1.3. In this case, the web server will be nginx with wolfSSL and the web client will be curl with wolfSSL.
+
+First, you will need to build the OpenQuantumSafe group's liboqs and their fork of OpenSSL to generate the certificate chain that uses the post-quantum FALCON signature scheme. Instructions for that are in wolfSSL git repoitory's INSTALL file. Note that when you generate your certificates, you will need to add your IP address as a subject alternative name. See here for more details: https://www.openssl.org/docs/manmaster/man5/x509v3_config.html
+
+When building wolfSSL, you will need to add a couple extra flags:
+
+```
+./configure --enable-nginx --with-liboqs --enable-all
+make all
+make check
+sudo make install
+```
+
+NOTE: `--enable-all` is for curl.
```

Comment:
Curl doesn't really need `--enable-all`. Can you narrow down the minimum viable set of build options?

Reply:
Sure.  Shouldn't be difficult.